sdk/dotnet/Pulumi/Config.cs

```diff
@@ -0,0 +1,196 @@
+// Copyright 2016-2018, Pulumi Corporation
```

Comment:
Is this the right copyright header we want?

This is what we use in our .ts files:

https://github.com/pulumi/pulumi/blob/06d4dde8898b2a0de2c3c7ff8e45f97495b89d82/sdk/nodejs/config.ts#L1-L13

Also, should these all be 2016-2019 (not 2018)?

Reply:
> Is this the right copyright header we want?

I would really hope not.  Basically so many other open source projects (like Roslyn) are perfectly fine with a single line copyright.  I can't imagine for a moment that we would have a problem with that line.  these large copyrights viscerally bother me.

Reply:
We should be consistent about the copyright headers if possible. E.g. presumably there's a reason we have the license in the header, though I don't know what that would be.